src/client/shader.cpp

```diff
@@ -730,6 +730,7 @@ ShaderInfo ShaderSource::generateShader(const std::string &name,
 	shaders_header << "#define ENABLE_WAVING_LEAVES " << g_settings->getBool("enable_waving_leaves") << "\n";
 	shaders_header << "#define ENABLE_WAVING_PLANTS " << g_settings->getBool("enable_waving_plants") << "\n";
 	shaders_header << "#define ENABLE_TONE_MAPPING " << g_settings->getBool("tone_mapping") << "\n";
+	shaders_header << "#define SATURATION_PARAM " << g_settings->getFloat("saturation_param")<< "\n";
```

Comment:
The `_PARAM` suffix is superfluous. Not complying with formatting rules (missing a space).